vars/createSha512.groovy

```diff
@@ -0,0 +1,8 @@
+Closure call() {
```

Comment:
This was a quick'n dirty method to pass around these functions calls - there might be a better way

Reply:
ELI5: what is the reason that we cannot use a normal `call` here?
From what I read it is meant for simplistic and able to passed as argument?
Therefore you can pass `createSha512` as an argument in promoteArtifacts?

Reply:
Not quite the ELI5 level, this is just copying prior art and convention.  `call(...)` is the method that is invoked when you do `createSha512(...)`.  Its signature supports parameters, typically we accept a map of values and stuff it into an object called args.  Typically we don't return anything, making the return type void.  So this function is following the same generalized pattern, but it doesn't accept parameters (we could add that) and it returns a closure that can be used.